src/WebJobs.Extensions.DurableTask/StandardConnectionStringProvider.cs

```diff
@@ -1,13 +1,15 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System;
```

Comment:
Yeah, exactly. We can remove it if we'd like, and if we keep it I may need to keep it in the dependency injection service container so that it's available.